### ISSUE TITLE
Is there a way to auto update the widgets?

### ISSUE THREAD

**Issue by saurabhsrivastava2009** [reportportal/reportportal#2029]
Hi Team,

Is there a way to auto-update the widgets with the latest data?
As of now, we have to do it manually.

Thanks in advance :)

**Comment by MaryiaRymarchyk** [comment]
Hi @saurabhsrivastava2009   , i'm not sure that i got your request in a right way.  Do you want to have updated widget on UI while new launches have been executed?
Actually we have two types of widget, let’s say common widgets  and widgets with materialized view.  Most of our widgets are common and they have automatic refresh each 60 seconds, also you can refresh each widget manually or reload browser page to have them updated.
There are only 2 widgets with materialized view: ‘Cumulative trend chart’ and ‘Component health check (table view)’. These widgets use temp tables and have huge background calculations, to avoid performance issues - we don’t refresh them automatically. Such kind of widgets can be updated only manually by clicking ‘Update’ button in the widget header, even browser refresh and re-login won’t update these widgets.  
To sum up, almost all widgets are updated automatically each minute, except 2: ‘Cumulative trend chart’ and ‘Component health check (table view)’.
Let me know if you have more questions here or the issue can be closed.

**Comment by saurabhsrivastava2009** [comment]
Thank you @MaryiaRymarchyk for the info :)
